apps/sms/views/shared/js/startup.js

```diff
@@ -108,16 +98,28 @@ var Startup = {
 
       MessageManager.init();
       Navigation.init();
+      InboxView.init();
```

Comment:
This changes flow for the activity case, previously we didn't call ```InboxView.init``` before lazy loading, but I believe it should not have big impact.

I just didn't want to call init in two palces :)